201-vmss-azureautomationdsc/webServer.ps1

```diff
@@ -0,0 +1,26 @@
+Configuration webServer
+{
+    Import-DSCResource -ModuleName PSDesiredStateConfiguration, xNetworking
+	
+	Node localhost
```

Comment:
A better name would be to call the configuration MyService and the node statement webserver, resulting in MyService.webserver. Localhost doesn't make much sense as a node configuraiton name.